### ISSUE TITLE
Importer doesn't find team

### ISSUE THREAD

**Issue by martinb3** [dutchcoders/slackarchive#16]
Hi there -- I'm using slackarchive and the importer, and I'm noticing this chunk of code always fails from importer.go:

```
	t := models.Team{}
	if err := i.db.Teams.FindId(team.ID).One(&t); err == nil {
		log.Debug("Team already exists: %s", t.ID)
	} else {
                // fails
		return nil, err
	}
```

What would create the team in the database? I can't find any team creation in the slackarchive or slackarchive-init projects.

**Comment by yanokwa** [comment]
Are you still having this problem, @martinb3?

**Comment by noisymime** [comment]
Not OP, but I'm still seeing the same issue on one of my imports

**Comment by yanokwa** [comment]
It sounds like some imports work and others don't. What's the difference between those imports?

**Comment by martinb3** [comment]
Yes, I ended up locally modifying it to work around this problem. I still could never find any reason the team should exist in the database before the other import steps. I suspect this line is fine on an existing import, but was never tested on an empty database.
